build.gradle.kts

```diff
@@ -499,6 +507,8 @@ tasks {
 
   buildPlugin {
     dependsOn(project.tasks.getByPath("buildCody"))
+    val get = composedJar.get()
```

Comment:
`get` does not sound like nice name ^^
Maybe just inline it?

Reply:
fixed